### PR TITLE
Remove non-ascii colon in JedisSentintelPool

### PR DESCRIPTION
**Pull request by NickFirmani**
This character will fail to compile unless compiled with the additional compiler option `-Dfile.encoding=UTF8`

### CODE REVIEW

File: src/main/java/redis/clients/jedis/JedisSentinelPool.java [hunk 1]
```diff
@@ -287,8 +287,8 @@ public void run() {
            */
           List<String> masterAddr = j.sentinelGetMasterAddrByName(masterName);  
           if (masterAddr == null || masterAddr.size() != 2) {
-            log.warn("Can not get master addr, master name: {}. Sentinel: {}：{}.",masterName,host,port);
-          }else{
+            log.warn("Can not get master addr, master name: {}. Sentinel: {}: {}.",masterName,host,port);
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/JedisSentinelPool.java]: why did you add a white space? where is none ascii?

Comment (NickFirmani) [reply src/main/java/redis/clients/jedis/JedisSentinelPool.java]: The colon is the non ascii character here, GitHub just renders it as such. Before `：` after `: `

File: src/main/java/redis/clients/jedis/JedisSentinelPool.java [hunk 2]
```diff
@@ -287,8 +287,8 @@ public void run() {
...
+          } else{
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/JedisSentinelPool.java]: Please remove wrongful and/or unnecessary spaces.

**Comment by sazzad16** [comment]
@gkorland We should not merge this right now, in https://github.com/xetorthio/jedis/pull/2062/files#diff-e6525ac1a247b59a49332a6584030684R290 it changes `host:port` to `host:<space>port`. This should not happen.

**Comment by sazzad16** [comment]
@gkorland Please see #2107

**Comment by gkorland** [comment]
@sazzad16 Ok I see your point, let's merge the other PR and close this one.
